From b6bba6873cd1e0f2b1a6256f5b928355fa811c17 Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Wed, 4 Sep 2019 21:11:48 +0200
Subject: [PATCH 01/10] Panic into recovery rather than bootloader

Getting last_kmsg/pstore from bootloader isn't possible for other people
than the OEM, but we have TWRP to access last_kmsg/pstore

Change-Id: If04bb6572dc66677d7b44f7d302b2d69ce526200
---
 init/reboot_utils.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/init/reboot_utils.cpp b/init/reboot_utils.cpp
index 76460a5df..5a12dd7af 100644
--- a/init/reboot_utils.cpp
+++ b/init/reboot_utils.cpp
@@ -34,7 +34,7 @@
 namespace android {
 namespace init {
 
-static std::string init_fatal_reboot_target = "bootloader";
+static std::string init_fatal_reboot_target = "recovery";
 static bool init_fatal_panic = false;
 
 void SetFatalRebootTarget() {
@@ -48,7 +48,7 @@ void SetFatalRebootTarget() {
     const char kRebootTargetString[] = "androidboot.init_fatal_reboot_target=";
     auto start_pos = cmdline.find(kRebootTargetString);
     if (start_pos == std::string::npos) {
-        return;  // We already default to bootloader if no setting is provided.
+        return;  // We already default to recovery if no setting is provided
     }
     start_pos += sizeof(kRebootTargetString) - 1;
 
-- 
2.30.2

